### PR TITLE
TASK: Adjust issue and pr templates

### PR DESCRIPTION
**Pull request by mficzel**
The templates now mention that the texts are used in change logs and explain the informations we need during review

Should be merged together with:
- https://github.com/neos/BuildEssentials/pull/61
- https://github.com/neos/flow-development-collection/pull/2837

### CODE REVIEW

**Comment by mficzel** [comment]
Target branch is 8.0 as this will shortly become the default branch.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Neos Forum
+    url: https://discuss.neos.io/
+    about: The official onlone forum for questions and discussions around Neos.
```
Comment (daniellienert) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: The official online forum for questions and discussions around Neos.
```

**Review by daniellienert — COMMENTED** [review]
NICE! Found a tiny typo

**Review by daniellienert — APPROVED** [review]

**Comment by crydotsnake** [comment]
That's awesome! Would this also be an idea for the neos-ui for example?

**Pull request merged**